content/bdd-tutorial.md

```diff
@@ -1,3 +1,37 @@
 ---
-title: BDD Tutorial
+title: Behaviour-Driven Development
 ---
+
+Behaviour-Driven development (or BDD) is a software development technique that encourages conversation and collaboration between developers, testers and business participants in a software project.
+
+BDD focuses on obtaining a clear and mutual understanding of desired behaviour of the application through conversations.
+It complements [TDD](#tdd) by writing test cases in a natural language that also non-programmers can read.
+The goal is to capture the outcome of the conversations in a meaningful way, using natural and domain language. The focus should be on
+the intended behaviour, not the implementation or technical details. It should create shared understanding, helping the developers to create the right things.
+
+> **"Behaviour-driven development is about implementing an application by describing its behavior from the perspective of its stakeholders”** -- Dan North
```

Comment:
The quote characters are different at the start and end of this blockquote.

Reply:
Thanks! Fixed it ;)